packages/application-shell/src/components/quick-access/quick-access.ctp.graphql

```diff
@@ -20,7 +20,6 @@ query QuickAccess(
   }
 
   productById: product(id: $searchText) @include(if: $canViewProducts) {
-    key
```

Comment:
We don't know why but without this field the apollo cache does not complain about a merging problem of product objects.

This is the only solution we found that would not result in having a warning. I checked the code and these `key` fields are actually not used.